source/enterprise/tech-guide/backup-eca-guide.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: Backup the Client Administration
+github_link: enterprise/tech-guide/backup-eca-guide.md
+indexed: false
+---
+
+This guide describes how you backup the Client Administration.
+
+<div class="toc-list"></div>
+
+## How to create a Backup
+
+### 1. The mysql database.
+
+First of all, you have to back up the mysql database.
+
+The database is called __shopware_client_administration__.
+
+You can do it e.g. with:
+````
+mysqldump -u [username] -p shopware_client_administration > [filename].sql
+````
+
+----------------------
+
+### 2. The Parameters.yml
+
+The second document to back up is the __parameters.yml__.
+
+It could be found in the folder:
+````
+client-Administration/components/ClientAdministration/app/config/parameters.yml
+````
+
+There you can find all hashes to encrypt and decrypt the values.
+
+Copy this file e.g:
+````
+cp parameters.yml parameters-copy.yml
+````
+
+----------------------
+
+### 3. The upload directory
+
+The last folder is the upload directory.
+
+````
+enterprise-dashboard/components/ClientAdministration/uploads
+````
+
+There are all hooks and each uploaded file.
+
+It is enough to copy it.
+````
+cp ./enterprise-dashboard/components/ClientAdministration/uploads uploadscopy.yml
+````
+
+----------------------
+
+## How to restore the backup
+
+### 1. The mysql database
+
+First you have to apply your sql file to the database.
+
+````
+create database shopware_client_administration;
```

Comment:
so this is SQL and we write SQL statements in all caps, also should not share one code element